README.md

```diff
@@ -2,39 +2,41 @@
 
 This SDK should provide you with all you'll need to start developing cloud-connected apps with the FeedHenry platform. The SDK provides access to cloud action calls, app authentication and authorization. 
 
-### Build
+## Build
 
-To build the SDK, you first need to install the Android SDK from http://developer.android.com/sdk/installing/index.html.
+### Prereqs
 
-Then you can clone the repo, go to the *sdk* directory, create a new file called local.properties file based on the local.properties.dist file, change the value of *sdk.dir* property to be the path to your local Android SDK directory.
+* [Java 6](http://www.oracle.com/technetwork/java/javase/downloads/index.html)
+* [Maven 3.1.1](http://maven.apache.org/)
+* Latest [Android SDK](https://developer.android.com/sdk/index.html) and [Platform version](http://developer.android.com/tools/revisions/platforms.html)
```

Comment:
When I tried this locally, I also need to make sure the ANDROID_HOME environment variable is set. Should add to the prereqs as well.